ci_tools/summarise_pyspelling.py

```diff
@@ -78,15 +78,17 @@ def find_all_words(file_path, search_word):
     if errors:
         all_words = set()
 
-        with open(os.path.join(os.path.dirname(__file__),'..','.dict_custom.txt'), encoding="utf-8") as d:
+        pyccel_folder = os.path.abspath(os.path.join(os.path.dirname(__file__),'..'))
+
+        with open(os.path.join(pyccel_folder,'.dict_custom.txt'), encoding="utf-8") as d:
             internal_dict = [w.strip() for w in d.readlines()]
 
         with open(args.output, 'w', encoding="utf-8") as f:
             print("There are misspelled words", file=f)
             for name, words in errors.items():
                 print("## `", name, "`", file=f)
                 for w in words:
-                    words_list = find_all_words(os.path.join("..",name.strip(":")), w)
+                    words_list = find_all_words(os.path.join(pyccel_folder,name.strip(":")), w)
```

Comment:
Would you mind adding an empty space after the commas? This is needed once in each of the 3 lines you modified, inside the call to `join`.

Reply:
Fixed [93140d4](https://github.com/pyccel/pyccel/pull/1451/commits/93140d47fbf43b86d413f50f8b116e56e1daead3)